packages/keystore_ffi/ios/libkeystore_ffi.a

```diff
@@ -0,0 +1 @@
+/Users/shadykhalifa/Code/sunshine-flutter/target/universal/debug/libkeystore_ffi.a
```

Comment:
same again

Reply:
same, when you run `cargo make` it would be overwritten with your local path.

Reply:
I guess we should add these to .gitignore

Reply:
I think that would make sense.